gio/Gir.toml

```diff
@@ -1364,6 +1364,10 @@ manual_traits = ["TlsConnectionExtManual"]
     name = "get_channel_binding_data"
     # Gir confuses the mutability of the data
     manual = true
+    [[object.function]]
```

Comment:
does it make sense to manually implement this function just for this? ideally someday we will fix gir to automatically generate proper code here..

Reply:
I would rather start changing the api for the users and as soon as gir is updated we can drop the manual impls